docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -942,3 +942,88 @@ spec:
   purpose: Informational
   tags: compliance, hunting
   contributors: nabilschear
+  ---
```

```suggestion
---
```

Seems to be some accidental indentation on this that breaks the yml!